h/static/scripts/annotator/guest.coffee

```diff
@@ -263,6 +279,17 @@ module.exports = class Guest extends Annotator
 
   onAnchorMousedown: ->
 
+  # Needed to get text selection working on iOS. Only used for touch screen devices.
+  userSelectionChanged: ->
+    # wait 500 ms after the last selection change event
+    if selectionEndTimeout
+      clearTimeout selectionEndTimeout
+    selectionEndTimeout = setTimeout((->
+      @Annotator.Util.getGlobal().trigger 'selectionEnd'
```

Comment:
Capture `Util = Annotator.Util` at the top of the file and then you can use it as `Util.getGlobal()`.